help/en/Acknowledgements.shtml

```diff
@@ -411,6 +411,9 @@
         System</a> tools for working with Dispatcher.
         </li>
 
+        <li>Dieter Flunkert, who contributed a decoder definition for the 
+            Decoderwerk LokComander.
```

Comment:
`LokComander` is misspelled. `LokCommander`

Reply:
It seems to have one 'm' or two depending on where you look.  I think 'mm' is probably right, unless this is a German form, so I changed it.  Thanks!